### PR TITLE
PP-7789 Breadcrumbs navigation header

### PR DESCRIPTION
**Pull request by sfount**
Update the service/ account navigation "phase_banner" using the new
breadcrumb according to the ticket specification.

Move the account specific navigation down to the line below.

Update pages that currently have a manual link back to the my services
page as they can now rely on the breadcrumb for that functionality.

All changes are behind the future method feature flag.

Beginning to formalising page types
* All services page
* Service page
* Account page 
* Top level

Removes the "My services" link from the navigation header in favour of the breadcrumbs, behind the future feature flag.

**All services page header**

Screenshot 2021-02-23 at 12 46 46

**Account page header**

Screenshot 2021-02-23 at 12 47 04

**Service page header**

Screenshot 2021-02-23 at 12 47 24

**Top level**
Screenshot 2021-02-23 at 12 47 57

with @stephencdaly

### CODE REVIEW

File: app/views/services/index.njk [hunk 1]
```diff
@@ -9,6 +9,10 @@
   <script src="/public/js/components/notification-banner.js"></script>
 {% endblock %}
 
+{% block beforeContent %}
```
Comment (kbottla) [inline app/views/services/index.njk:12]: Is this supposed to have anything here or is it intended ?

File: app/views/payouts/list.njk [hunk 2]
```diff
@@ -1,17 +1,25 @@
+{% from '../macro/breadcrumbs.njk' import breadcrumbs %}
 {% extends "../layout.njk" %}
 
 {% block pageTitle %}
 Payments to your bank account - GOV.UK Pay
 {% endblock %}
 
 {% block beforeContent %}
-  {{ super() }}
+  {% if enabledMyServicesAsDefaultView %}
```
Comment (kbottla) [inline app/views/payouts/list.njk:9]: Shall we change this similar to all service transactions with live/test mode https://github.com/alphagov/pay-selfservice/blob/73d8712ab809bdc2b9125ef59d2e8d5fcc7b307e/app/views/transactions/index.njk#L31-L45

File: app/views/services/index.njk (hunk 1 above)
Comment (sfount) [reply app/views/services/index.njk:12]: It's intended, the extension mechanism in Nunjucks means that this template will by default use the `beforeContent` inherited from the main layout. As this page implements its own unique behaviour (the one page without any kind of header) it overrides this

File: app/views/payouts/list.njk (hunk 2 above)
Comment (kbottla) [reply app/views/payouts/list.njk:9]: As discussed, that can be a separate as changes are behind feature flag

**Review by kbottla — APPROVED** [review]

File: app/views/payouts/list.njk (hunk 2 above)
Comment (sfount) [reply app/views/payouts/list.njk:9]: 👍  great spot, I'll push the tag for breadcrumbs on payouts as separate PR

**Pull request merged**